### PR TITLE
Domains

### PR DESCRIPTION
**Pull request by robertsteilberg**
* changes domain service urls to `api/v3/domains`

### CODE REVIEW

**Comment by drewctate** [comment]
Has this been tested on Portal @robertsteilberg?

**Comment by robertsteilberg** [comment]
No, but Portal does not use the domain service or ever encounter the domain service, so I do not think it will make a difference

**Review by drewctate — APPROVED** [review]
I think you're probably safe. As you say, these utils shouldn't even be fired on Portal. Nice work.

File: packages/sites/src/get-domain.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export function getDomains(
   requestOptions?: IRequestOptions
 ): Promise<any> {
   const apiUrl = getHubApiUrl(requestOptions);
-  const url = `${apiUrl}/utilities/domains`;
+  const url = `${apiUrl}/api/v3/domains`;
```
Comment (tomwayson) [inline packages/sites/src/get-domain.ts:18]: why not use `_getDomainServiceUrl()` here/

File: packages/sites/src/domains/add-domain.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export function addDomain(
   }
   const headers = _getAuthHeader(hubRequestOptions);
   headers["Content-Type"] = "application/json";
-  const url = `${_getDomainServiceUrl(hubRequestOptions)}?force=true`; // Use the force...
+  const url = `${_getDomainServiceUrl(hubRequestOptions)}`;
```
Comment (drewctate) [inline packages/sites/src/domains/add-domain.ts:23]: Just curious. Why no `force=true`?

File: .gitignore [hunk 3]
```diff
@@ -84,3 +84,6 @@ packages/*/debug/
 
 ## for john to poke at
 test.html
+
+.idea/
+.vscode/
```
[the remark this answers is not in the document]
Comment (robertsteilberg) [reply .gitignore:89]: ;)

File: packages/sites/src/get-domain.ts (hunk 1 above)
Comment (robertsteilberg) [reply packages/sites/src/get-domain.ts:18]: good point i'll refactor that

File: packages/sites/src/domains/add-domain.ts (hunk 2 above)
Comment (robertsteilberg) [reply packages/sites/src/domains/add-domain.ts:23]: this flag never worked in the original domain service, so we didn't implement it in the new one

File: packages/sites/src/get-domain.ts (hunk 1 above)
Comment (robertsteilberg) [reply packages/sites/src/get-domain.ts:18]: looks like a type mismatch--`_getDomainServiceUrl` expects `IHubRequestOptions` which we don't have in `getDomains`

Comment (tomwayson) [reply packages/sites/src/get-domain.ts:18]: you can change that to accept `hubRequestOptions: IHubRequestOptions | hubApiUrl: string`

**Review by tomwayson — APPROVED** [review]
thanks @robertsteilberg

**Pull request merged**